src/main/java/frc/robot/utilities/LEDSubStrip.java

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+// THis class is a wrapper on AddressableLEDBuffer that allows logical divisions of one continuous
+// strip of LEDs
+
+package frc.robot.utilities;
+
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+
+public class LEDSubStrip {
+  private int begin;
+  private int end;
+  private boolean direction;
+  private AddressableLEDBuffer stripBuffer;
```

Comment:
Nit - you could consider making all of these `final` since they really should not change once the `LEDSubStrip` instance has been created.